diff -Naur supertux-0.3.4.orig/CMakeLists.txt supertux-0.3.4/CMakeLists.txt
--- supertux-0.3.4.orig/CMakeLists.txt	2014-04-03 22:13:02.000000000 +0400
+++ supertux-0.3.4/CMakeLists.txt	2014-04-03 22:14:06.000000000 +0400
@@ -106,6 +106,23 @@
 #INCLUDE_DIRECTORIES(${ICONV_INCLUDE_DIR})
 #LINK_LIBRARIES(${ICONV_LIBRARY})
 
+find_library(SQUIRRELL_LIB squirrel
+             PATH_SUFFIXES squirrel )
+find_library(SQUIRRELL_STDLIB sqstdlib
+             PATH_SUFFIXES squirrel )
+find_path(SQUIRREL_INCLUDE_DIR squirrel.h PATH_SUFFIXES squirrel)
+INCLUDE_DIRECTORIES(${SQUIRREL_INCLUDE_DIR})
+
+if (SQUIRRELL_LIB AND SQUIRRELL_STDLIB)
+  message (STATUS "SQUIRRELL_LIB    = ${SQUIRRELL_LIB}")
+  message (STATUS "SQUIRRELL_STDLIB = ${SQUIRRELL_STDLIB}")
+else()
+  message (STATUS "SQUIRRELL_LIB    = ${SQUIRRELL_LIB}")
+  message (STATUS "SQUIRRELL_STDLIB = ${SQUIRRELL_STDLIB}")
+  message (FATAL_ERROR "squirrel libraries not found")
+endif()
+
+
 ## Find revision of WC
 MARK_AS_ADVANCED(SVNVERSION_EXECUTABLE)
 ##FIND_PROGRAM(SVNVERSION_EXECUTABLE svnversion)
@@ -150,17 +167,17 @@
 
 ## Also execute instructions in external/squirrel/CMakeLists.txt
 
-ADD_SUBDIRECTORY(${CMAKE_CURRENT_SOURCE_DIR}/external/squirrel)
+#ADD_SUBDIRECTORY(${CMAKE_CURRENT_SOURCE_DIR}/external/squirrel)
 
 ## Add squirrel lib dir to search path
 
-LINK_DIRECTORIES(${CMAKE_CURRENT_SOURCE_DIR}/external/squirrel)
+#LINK_DIRECTORIES(${CMAKE_CURRENT_SOURCE_DIR}/external/squirrel)
 
 ## Some additional include paths
 
 include_directories (${CMAKE_BINARY_DIR}/)
 include_directories (${CMAKE_CURRENT_SOURCE_DIR}/src/)
-include_directories (${CMAKE_CURRENT_SOURCE_DIR}/external/squirrel/include/)
+#include_directories (${CMAKE_CURRENT_SOURCE_DIR}/external/squirrel/include/)
 include_directories (${CMAKE_CURRENT_SOURCE_DIR}/external/tinygettext/)
 include_directories (${CMAKE_CURRENT_SOURCE_DIR}/external/findlocale/)
 include_directories (${CMAKE_CURRENT_SOURCE_DIR}/external/obstack/)
@@ -341,7 +358,7 @@
 
 ## Link supertux binary with squirrel and other libraries
 
-TARGET_LINK_LIBRARIES(supertux2 squirrel)
+TARGET_LINK_LIBRARIES(supertux2 ${SQUIRRELL_LIB} ${SQUIRRELL_STDLIB})
 TARGET_LINK_LIBRARIES(supertux2 ${SDL_LIBRARY})
 TARGET_LINK_LIBRARIES(supertux2 ${SDLIMAGE_LIBRARY})
 TARGET_LINK_LIBRARIES(supertux2 ${OPENAL_LIBRARY})
