--- SuperTux-v0.5.1-Source/CMakeLists.txt.omv~	2016-11-07 12:21:16.846545835 +0100
+++ SuperTux-v0.5.1-Source/CMakeLists.txt	2016-11-07 12:27:59.938570499 +0100
@@ -318,6 +318,22 @@ endif()
 INCLUDE(ConfigureChecks)
 
 
+OPTION(USE_SYSTEM_SQUIRREL "Use system squirrel libraries" ON)
+if(USE_SYSTEM_SQUIRREL)
+	find_library(SQUIRRELL_LIB squirrel PATH_SUFFIXES squirrel)
+	find_library(SQUIRRELL_STDLIB sqstdlib PATH_SUFFIXES squirrel)
+	find_path(SQUIRREL_INCLUDE_DIR squirrel.h PATH_SUFFIXES squirrel)
+	INCLUDE_DIRECTORIES(${SQUIRREL_INCLUDE_DIR})
+
+	if (SQUIRRELL_LIB AND SQUIRRELL_STDLIB)
+		message (STATUS "SQUIRRELL_LIB    = ${SQUIRRELL_LIB}")
+		message (STATUS "SQUIRRELL_STDLIB = ${SQUIRRELL_STDLIB}")
+	else()
+		message (STATUS "SQUIRRELL_LIB    = ${SQUIRRELL_LIB}")
+		message (STATUS "SQUIRRELL_STDLIB = ${SQUIRRELL_STDLIB}")
+		message (FATAL_ERROR "squirrel libraries not found")
+	endif()
+else()
 ## Also build external/squirrel
 
 IF(NOT EXISTS ${CMAKE_CURRENT_SOURCE_DIR}/external/squirrel/CMakeLists.txt)
@@ -352,6 +368,9 @@ ELSE()
 ENDIF()
 
 INCLUDE_DIRECTORIES(SYSTEM ${SQUIRREL_PREFIX}/include)
+set(SQUIRREL_LIB squirrel_lib)
+set(SQUIRREL_STDLIB sqstdlib_lib)
+endif()
 
 ## Also build external/tinygettext
 
@@ -611,8 +630,8 @@ TARGET_INCLUDE_DIRECTORIES(supertux2_lib
 TARGET_LINK_LIBRARIES(supertux2_lib PUBLIC ${SDL2_LIBRARIES})
 TARGET_LINK_LIBRARIES(supertux2_lib PUBLIC ${SDL2IMAGE_LIBRARIES})
 
-TARGET_LINK_LIBRARIES(supertux2_lib PUBLIC squirrel_lib)
-TARGET_LINK_LIBRARIES(supertux2_lib PUBLIC sqstdlib_lib)
+TARGET_LINK_LIBRARIES(supertux2_lib PUBLIC ${SQUIRREL_LIB})
+TARGET_LINK_LIBRARIES(supertux2_lib PUBLIC ${SQUIRREL_STDLIB})
 TARGET_LINK_LIBRARIES(supertux2_lib PUBLIC tinygettext_lib)
 TARGET_LINK_LIBRARIES(supertux2_lib PUBLIC sexp)
 TARGET_LINK_LIBRARIES(supertux2_lib PUBLIC ${OPENAL_LIBRARY})
@@ -781,7 +800,11 @@ configure_file(data/levels/misc/menu.stl
 ## CPack/Installation-specific stuff
 
 INCLUDE(InstallRequiredSystemLibraries)
+if(USE_SYSTEM_SQUIRREL)
+SET(DIRS ${Boost_LIBRARY_DIRS} ${CMAKE_CURRENT_BINARY_DIR}/external/tinygettext)
+else()
 SET(DIRS ${Boost_LIBRARY_DIRS} ${CMAKE_CURRENT_BINARY_DIR}/external/squirrel ${CMAKE_CURRENT_BINARY_DIR}/external/tinygettext)
+endif()
 
 IF(${CMAKE_SYSTEM_NAME} MATCHES "Darwin" AND NOT DISABLE_CPACK_BUNDLING)
     SET(INFOPLIST_CFBUNDLEEXECUTABLE "SuperTux")
